db/compaction_iterator.cc

```diff
@@ -188,6 +210,11 @@ void CompactionIterator::NextFromInput() {
       current_user_key_sequence_ = kMaxSequenceNumber;
       current_user_key_snapshot_ = 0;
 
+      if (compaction_listener_)
+        compaction_listener_->OnCompaction(compaction_->level(), ikey_.user_key,
+                                           fromInternalValueType(ikey_.type),
+                                           value_, ikey_.sequence, true);
```

Comment:
In Google C++ Style we are following:
"In general, curly braces are not required for single-line statements, but they are allowed if you like them; conditional or loop statements with complex conditions or statements may be more readable with curly braces. Some projects require that an if must always always have an accompanying brace."

I think this statement is complicated so I suggest we add braces. In fact in this code base we always follow that is must have a brace. So if it is possible, I hope we add it in all the places.